src/main/kotlin/blackJack/domain/Player.kt

```diff
@@ -1,25 +1,7 @@
 package blackJack.domain
 
-class Player(val name: String?) {
+class Player(name: String) : People(name) {
     init {
-        checkName()
+        require(!name.isBlank()) { "이름은 없을수 받을수 없습니다." }
```

Comment:
`String`에 대해 호출할 수 있는 `isNotEmpty()` 또는 `isNotBlank()`가 있습니다.

```suggestion
        require(name.isNotBlank()) { "이름은 없을수 받을수 없습니다." }
```